servicecall/services/src/main/resources/application.properties

```diff
@@ -15,15 +15,19 @@
 ## limitations under the License.
 ## ---------------------------------------------------------------------------
 
-# Spring Boot
-management.endpoints.enabled-by-default=false
-management.endpoint.health.enabled=true
-
-# Camel
-camel.springboot.main-run-controller=true
-camel.springboot.jmx-enabled=false
-
-# expose actuator endpoint via HTTP
-management.endpoints.web.exposure.include=camelroutes,jolokia,metrics,prometheus,health
-# turn on actuator health check
-management.endpoint.health.enabled = true
\ No newline at end of file
+- route:
```

Comment:
yaml in a properties file?

Reply:
Fixed, reverted to previous application.properties file